client/state/ui/layout-focus/reducer.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import includes from 'lodash/includes';
+
+/**
+ * Internal dependencies
+ */
+import {
+	LAYOUT_SET_FOCUS,
+	LAYOUT_NEXT_FOCUS_ACTIVATE,
+	LAYOUT_SET_NEXT_FOCUS,
+	SERIALIZE,
+	DESERIALIZE,
+} from 'state/action-types';
+import layoutFocusModule from 'lib/layout-focus';
+
+const initialState = { current: null, previous: null, next: null };
+
+export default function layoutFocus( state = initialState, action ) {
+	const validAreas = [ 'content', 'sidebar', 'sites', 'preview' ];
+	const isValidArea = area => includes( validAreas, area );
+	switch ( action.type ) {
+		case LAYOUT_SET_FOCUS:
+			if ( ! isValidArea( action.area ) ) {
+				return state;
+			}
+			// Temporary while replacing module with this reducer
+			layoutFocusModule.set( action.area );
```

Comment:
Yeah, I don't love it, but it was the easiest way to keep parity during transition, and as you say, it's already everywhere 😝 

That said, happy to hear other ideas!